iml-agent/src/poller.rs

```diff
@@ -83,6 +83,7 @@ pub async fn create_poller(
                         sessions.reset_empty(&name)
                     });
 
+                    #[allow(unused_must_use)]
                     tokio::spawn(async {
                         fut.await;
```

Comment:
`fut.map(drop).await;`

The future is already carrying an `Err(())` at this point. See line 83 above.

Reply:
Done